tests_integration/nativeapp/test_validate.py

```diff
@@ -0,0 +1,45 @@
+import os
+import uuid
+
+from snowflake.cli.api.project.util import generate_user_env
+from tests.project.fixtures import *
+from tests_integration.test_utils import (
+    pushd,
+)
+
+USER_NAME = f"user_{uuid.uuid4().hex}"
+TEST_ENV = generate_user_env(USER_NAME)
+
+
+@pytest.mark.integration
+def test_nativeapp_validate(runner, snowflake_session):
```

Comment:
For sake of completeness, we should add one more test on a failing script.

Reply:
added